k8s.io/README.md

```diff
@@ -9,7 +9,7 @@ Vanity URL(s)
 |  | k8s.io | kubernetes.io |
 | --- | --- | --- |
 | APT downloads| https://apt.k8s.io | https://apt.kubernetes.io |
-| Blog | https://blog.k8s.io | |
+| Blog | https://blog.k8s.io | https://blog.kubernetes.io |
```

Comment:
This should be https://kubernetes.io/blog/

Reply:
That's true! then also blog.k8s.io need to k8s.io/blog. right?

Reply:
yes